modules/tinymce/src/core/main/ts/api/html/Styles.ts

```diff
@@ -298,9 +272,6 @@ const Styles = (settings?: StylesSettings, schema?: Schema): Styles => {
               value = value.toLowerCase();
             }
 
-            // Convert RGB colors to HEX
```

Comment:
Interesting so we are switching from hex to rgb as the default. I though we where only getting rid of public apis. This also means that the colors will not be uniform anymore they will be mixed hex and rgb.

Reply:
We're not changing anything to rgb, we're just no longer converting everything to hex.